CHANGELOG.md

```diff
@@ -7,11 +7,26 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
-## [4.9.1](https://github.com/ASFHyP3/hyp3-gamma/compare/v4.9.0...v4.9.1)
+## [5.0.0](https://github.com/ASFHyP3/hyp3-gamma/compare/v4.9.0...v5.0.0)
 
-### Changed
-* InSAR readme and xml templates have been updated to reflect the change in displacement map options and to include more information on adjusting unwrapped phase and displacement map values relative to a user-selected reference point.
-* Updated the URL for the [Copernicus DEM](https://spacedata.copernicus.eu/explore-more/news-archive/-/asset_publisher/Ye8egYeRPLEs/blog/id/434960) information webpage.
+### Added
+* An `--include-displacement-maps` argument to the `insar` entrypoint which will cause both the
+  line-of-sight and vertical displacement maps to be included.
+
+### Changed
+* The `include_los_displacement` parameter to `insar_sentinel_gamma` in `hyp3_gamma.insar.ifm_sentinel`
+  has changed to `include_displacement_maps`. Now, both the line-of-sight displacement and the vertical
+  displacement maps will be included if this option is set to `True`.
+* InSAR README and XML templates have been updated to reflect the change in displacement map options
+  and to include more information on adjusting unwrapped phase and displacement map values relative
+  to a user-selected reference point.
+* Updated the URL for the [Copernicus DEM](https://spacedata.copernicus.eu/explore-more/news-archive/-/asset_publisher/Ye8egYeRPLEs/blog/id/434960)
+  information webpage.
+
+### Deprecated
+* The `--include-los-displacment` argument to the `insar` entrypoint is depreciated and may be removed
```

Comment:
Should depreciated be changed to deprecated to match the header?

Reply:
yes, yes it should.